docs/tutorials/getting-started-with-node-js.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Getting Started with Node JS
+group: software-languages
+author: "@liam-b"
+---
+
+{% include /style/icon.html type="warning" %}
+Note that Node JS run really slowly on the EV3 brick and is limited to NPM version 0.10.29 which can be a problem when trying to install some libraries.
+{: .alert .alert-warning}
+
+**Before you start, make sure that you have configured a network connection to
+your ev3dev device and have opened an SSH connection to it.**
+
+## Installing ev3dev-lang
+To install the library run:
```

Comment:
Without an explicit mention that this should be run via SSH, some people will try running this on their PC :laughing: